content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -52,13 +52,11 @@ The upgrade workflow at high level is the following:
     {{< tabs name="k8s_install_versions" >}}
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
     apt update
-    apt-cache madison kubeadm
-    # find the latest 1.17 version in the list
+    apt-cache madison kubeadm | awk '{print $3}' | grep -m1 ^1.17
```

Comment:
the comment above the line is still useful:

> # find the latest 1.17 version using apt-cache

Reply:
sure